python_modules/dagster/dagster/grpc/impl.py

```diff
@@ -347,13 +352,16 @@ def get_external_execution_plan_snapshot(recon_pipeline, args):
             else recon_pipeline
         )
 
+        instance = DagsterInstance.from_ref(args.instance_ref) if args.instance_ref else None
```

Comment:
hmmmm this assumes that the DagsterInstance is always loadable in every gRPC server which I don't think is always the case unfortunately :/ I don't think we can have this in the critical path to launch a run